packages/grid/x-data-grid/src/internals/index.ts

```diff
@@ -49,6 +49,7 @@ export {
   preferencePanelStateInitializer,
 } from '../hooks/features/preferencesPanel/useGridPreferencesPanel';
 export { useGridEditing, editingStateInitializer } from '../hooks/features/editing/useGridEditing';
+export * from '../hooks/features/editing/gridEditingSelectors';
```

Comment:
I tend to avoid the "export all" pattern in the `internal` to make keep clear what we are exporting as private and prioritize public imports.